README.md

```diff
@@ -30,4 +30,23 @@ Describe how to run the project, e.g.:
 
 ```bash
 poetry run python src/main.py
-```
\ No newline at end of file
+```
+
+# Downloading Data
+
+Download data from [here](https://drive.google.com/open?id=1bTM5eh9wQ4U8p2ANOGbhZqTvDOddFnlI) ~19G in main directory in
+repository
+or use the following command in main directory in repository:
+
+```bash
+wget -O void_1500.zip https://drive.google.com/uc?id=1bTM5eh9wQ4U8p2ANOGbhZqTvDOddFnlI
```

Comment:
To polecenie zwraca html zamiast pobierać paczkę z datasetem:
```html
<!DOCTYPE html><html><head><title>Google Drive - Virus scan warning</title><meta http-equiv="content-type" content="text/html; charset=utf-8"/><style nonce="WXnd411bNpbNDoaF_fgphA">.goog-link-button{position:relative;color:#15c;text-decoration:underline;cursor:pointer}.goog-link-button-disabled{color:#ccc;text-decoration:none;cursor:default}body{color:#222;font:normal 13px/1.4 arial,sans-serif;margin:0}.grecaptcha-badge{visibility:hidden}.uc-main{padding-top:50px;text-align:center}#uc-dl-icon{display:inline-block;margin-top:16px;padding-right:1em;vertical-align:top}#uc-text{display:inline-block;max-width:68ex;text-align:left}.uc-error-caption,.uc-warning-caption{color:#222;font-size:16px}#uc-download-link{text-decoration:none}.uc-name-size a{color:#15c;text-decoration:none}.uc-name-size a:visited{color:#61c;text-decoration:none}.uc-name-size a:active{color:#d14836;text-decoration:none}.uc-footer{color:#777;font-size:11px;padding-bottom:5ex;padding-top:5ex;text-align:center}.uc-footer a{color:#15c}.uc-footer a:visited{color:#61c}.uc-footer a:active{color:#d14836}.uc-footer-divider{color:#ccc;width:100%}.goog-inline-block{position:relative;display:-moz-inline-box;display:inline-block}* html .goog-inline-block{display:inline}*:first-child+html .goog-inline-block{display:inline}sentinel{}</style><link rel="icon" href="//ssl.gstatic.com/docs/doclist/images/drive_2022q3_32dp.png"/></head><body><div class="uc-main"><div id="uc-dl-icon" class="image-container"><div class="drive-sprite-aux-download-file"></div></div><div id="uc-text"><p class="uc-warning-caption">Google Drive can't scan this file for viruses.</p><p class="uc-warning-subcaption"><span class="uc-name-size"><a href="/open?id=1bTM5eh9wQ4U8p2ANOGbhZqTvDOddFnlI">void_1500.zip</a> (19G)</span> is too large for Google to scan for viruses. Would you still like to download this file?</p><form id="download-form" action="https://drive.usercontent.google.com/download" method="get"><input type="submit" id="uc-download-link" class="goog-inline-block jfk-button jfk-button-action" value="Download anyway"/><input type="hidden" name="id" value="1bTM5eh9wQ4U8p2ANOGbhZqTvDOddFnlI"><input type="hidden" name="confirm" value="t"><input type="hidden" name="uuid" value="d72c4843-2d98-4374-bc46-7fb8dcd86623"></form></div></div><div class="uc-footer"><hr class="uc-footer-divider"></div></body></html>
```

Reply:
done